src/main/java/net/snowflake/client/core/SFStatement.java

```diff
@@ -397,6 +398,20 @@ public Object executeHelper(
                   IncidentUtil.oneLiner("Failed to upload binds " + "to stage:", ex)),
               null,
               requestId);
+        } catch (SQLException ex) {
+          logger.debug(
+              "Exception encountered trying to upload binds to stage. Attaching binds in payload instead. ",
+              ex);
```

Comment:
Can we add more context here which indicates the exception is for stream upload?

Reply:
Yes, added comment that it's for streaming upload.